packages/calypso-build/webpack.config.js

```diff
@@ -0,0 +1,107 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const DuplicatePackageCheckerPlugin = require( 'duplicate-package-checker-webpack-plugin' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const Minify = require( '@automattic/calypso-build/webpack/minify' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+const wordpressExternals = require( '@automattic/calypso-build/webpack/wordpress-externals' );
+
+/**
+ * Internal dependencies
+ */
+// const { workerCount } = require( './webpack.common' ); // todo: shard...
+
+/**
+ * Internal variables
+ */
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+/**
+ * Return a webpack config object
+ *
+ * @return {object} webpack config
+ */
+function getWebpackConfig(
+	env,
+	{
+		entry,
+		'output-path': outputPath = path.join( __dirname, 'dist' ),
+		'output-filename': outputFilename = '[name].js',
+		'output-libary-target': outputLibraryTarget = 'window',
+	}
```

Comment:
This choice of parametrization is supposed to organically tap into [Webpack's way of accepting command line args](https://webpack.js.org/api/cli/#output-options). We should be able to use this config file with Webpack's CLI, and override these args, e.g. `webpack --config ./webpack.config.js --output-filename="[chunk].js"`.

(Meaning I'm trying to essentially re-use Webpack's API without adding any new constraints.)